lib/Controller/SignalingController.php

```diff
@@ -473,15 +473,17 @@ private function backendPing(array $request): DataResponse {
 			]);
 		}
 
+		$pingSessionIds = [];
 		$now = $this->timeFactory->getTime();
 		foreach ($request['entries'] as $entry) {
-			if (array_key_exists('userid', $entry)) {
-				$room->ping($entry['userid'], $entry['sessionid'], $now);
-			} else {
-				$room->ping('', $entry['sessionid'], $now);
+			if ($entry['sessionid'] !== '0') {
+				$pingSessionIds[] = $entry['sessionid'];
```

Comment:
So now we don't ping for each user... Is that fine? Like the update if the userid exists is different...

Reply:
well before we also pinged users which didnt have a session, but if you dont have a session we dont return you as acitve user anyway anymore, so this is fine.

Reply:
:rocket: